### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by joshuagl**
Some minor changes for round 4.
- Addresses #61 by stating pip should add an option and suggesting `--unsafely-disable-package-verification`.
- Adds additional acknowledgements to address #58

### CODE REVIEW

File: pep-0458.txt [hunk 1]
```diff
@@ -263,9 +263,9 @@ example, TUF MAY be integrated with the pip package manager. Thus, new versions
 of pip going forward SHOULD use TUF by default to download and verify packages
 from PyPI before installing them. However, there may be unforeseen issues that
 might prevent users from installing or updating packages, including pip itself,
-via TUF. Therefore, pip MAY provide an undocumented option
-(e.g., `--no-pep-458`) in order to work around such issues until they are
-resolved.
+via TUF. Therefore, pip SHOULD provide an option (e.g.,
+`--unsafely-disable-package-verification`) in order to work around such issues
```
Comment (trishankatdatadog) [inline pep-0458.txt]: @jhdalek55 What do you think about `--unsafely-disable-package-verification`? Can you think of something more succinct?

**Review by trishankatdatadog — APPROVED** [review]

File: pep-0458.txt (hunk 1 above)
Comment (jhdalek55) [reply pep-0458.txt]: It is a bit wordy. Also it doesn't suggest this is a temporary or contingency option, which if I understand this text snippet, is what this proposes.  Does "package" have to be in the phrase?  I'm thinking "stall-verification" would work, or if the word "package" is needed to distinguish it from other verification processes, maybe "stall-package-verification." The word stall implies temporary status, and the phrase as a whole is at least one word shorter.

Comment (joshuagl) [reply pep-0458.txt]: I deliberated on this for a bit and ultimately decided that wordy and explicit was better - the action is unsafe and not generally recommended, it disables package verification.

I'm not sure temporary is the right word, the option would disable package verification for that invocation of pip but the unverified packages that are installed will persist on the system permanently (until updated or removed).

Thinking out loud here, does it also make sense to suggest an option for pip to undo unsafe usage, i.e. re-download and verify all installed packages? AFAIK pip doesn't store packages once they are installed, so this would require re-fetching any installed packages.

Comment (jhdalek55) [reply pep-0458.txt]: I agree with you that clarity is probably the most important thing in labeling this option, so as ungainly as the phrase seems, you are probably correct in going forward with what you have.

Comment (trishankatdatadog) [reply pep-0458.txt]: @joshuagl To answer your question about undoing unsafe usage: well, once a package has been installed unsafely, it's hard to really undo any damage unless nuked from orbit. Probably not worth the effort, but just my 0.02 BTC.

Comment (joshuagl) [reply pep-0458.txt]: I am erring on the side of "probably not worth the effort" too - I anticipate it would require significant changes to Pip and the outcome can be achieved in different ways (i.e. venv and nuke/replace that if you've had to use `--unsafely-disable-package-verification`)

Comment (lukpueh) [reply pep-0458.txt]: Besides `--disable-*`, pip also uses `--force-*`, which might make a more serious impression, e.g.  `--force-unsafe-install`.

**Review by lukpueh — APPROVED** [review]
